### ISSUE TITLE
Orks: Outdated points for Big Mek w/Kustom Force Field

### ISSUE THREAD

**Issue by dplummer** [BSData/wh40k-9e#7904]
**File/Catalogue:** Orks.cat

**Description:**

tldr: Big Mek w/KFF points should be 60, not 75, because of publication date ordering.

There are 4 publications that can be consulted for the points cost for the Big Mek w/Kustom Force Field from Saga of the Beast:

* Saga of the Beast 3/2020: 55points
* Saga of the Beast FAQ 8/2020: 75pts
* Munitorum Field Manual 7/2020: 60points
* Munitorum Field Manual FAQ 8/2020: Not listed

The Saga of the Beast FAQ changes the wording in Saga of the Beast to 75pts. The Munitorum Field Manual specifically says "The points values listed in this book replace any published previously, [...]" (pg 3). As Saga of the Beast was published before the MFM, the points value listed in MFM takes precedence. Therefore, the points value for Big Mek w/Kustom Force Field should be 60pts, not 75pts.

Similarly confusing, but not the issue at hand:

If the date of the FAQ means the Saga FAQ supersedes the MFM - then which power level is used? In the Saga FAQ the Power Level is listed as 4, but in the Power Level Update 2020 it is listed as 5. When sorting by publication date the Saga FAQ is newer than the Power Level Update.

As an aside - the Power level for the Big Mek w/KFF was added as 5PL - which at the standard 20pts per PL is just so far away from what's listed as to be very confusing. Perhaps the mistake is that the KFF should be listed as wargear in Saga (+20pts) *and* the Saga FAQ should be added to the MFM FAQ, making the total points 95pts.

**Comment by williamcaw** [comment]
it has already been done man just refresh the data

**Comment by dplummer** [comment]
> it has already been done man just refresh the data

In the current release 9.0.11, and in the master branch data file, it's 75pts. I'm arguing that it should be 60pts.

**Comment by williamcaw** [comment]
@Mad-Spy

**Comment by dplummer** [comment]
I just read through https://github.com/BSData/wh40k/issues/7671. I see you've already looked at this issue. I don't agree with the interpretation, but understand the reasoning. You're welcome to close this if I didn't change your mind @Mad-Spy. Thanks. I wish GW was clearer 🤷

**Comment by tag8833** [comment]
SotB FAQ is newer than MFM, so I think that takes precedence.  

On the other hand, the MFM Power Rating Update probably has the advantage over SotB FAQ.   

I think it should be 75 pts, and 5 PR, which is dumb, and probably not RAI, but is the closest I can derive for RAW.   

Anyone disagree?

**Comment by dplummer** [comment]
SotB FAQ is newer than the MFM Power Rating Update (sort by Recently Updated and Saga is newest), so I'd argue 4PL. IMO the options are either 4PL 75pts or 5PL 60pts.

**Comment by tag8833** [comment]
Fair enough.   75 / 20 is closer to 4 than 60 / 20 is to 5, so we are going with that until GW clarifies.   

However, I respect the confusion, so maybe I should make it an option for those with a different read on the situation:

Unit Upgrade for Big Mek w/ KFF 
PR Confusion:  ____  4 (SotB FAQ)  or   ____ 5 (MFM FAQ)
Pts Confusion:  ____ 75 (SotB FAQ) or   ____ 60 (MFM)

Not sure if that makes it better or worse.

**Comment by DrTobogganMD** [comment]
Fixed per Author
